kvmagent/kvmagent/test/vm_plugin_testsuite/test_check_vm_xml_defaultvalue.py

```
@@ -24,7 +24,7 @@ class TestVmXmLDefaultValue(TestCase, vm_utils.VmPluginTestStub):
     def setUpClass(cls):
         network_utils.create_default_bridge_if_not_exist()
 
-    @misc.test_for(handlers=[
+    @env.test_for(handlers=[
```

Ruff (0.8.2)

`env` may be undefined, or defined from star imports

(F405)